src/roswire/ros2/launch.py

```diff
@@ -33,33 +33,11 @@ def for_app_instance(cls,
                          ) -> 'ROS2LaunchManager':
         return ROS2LaunchManager(app_instance=app_instance)
 
-    def read(self,
-             filename: str,
-             *,
-             package: Optional[str] = None,
+    def read(self, filename: str,
```

Comment:
#406 Defer to this issue

Reply:
This is no longer a behavioural subtype of `LaunchFileManager`. The API for the `read` method should be the same for `LaunchFileManager`, `ROS1LaunchFileManager`, and `ROS2LaunchFileManager`. Why the change in API?

Reply:
Hmm. I don't think I did. I think this API is from the original ROS2LaunchFileManager I started with. I've changed it to make it consistent.

Reply:
It looks like the `package` parameter is missing, and `argv` is no longer a keyword-only parameter.